roles/satellite-clone/tasks/main.yml

```diff
@@ -3,6 +3,16 @@
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: check satellite version from the config files
+  command: tar zxf {{ backup_dir }}/config_files.tar.gz  etc/foreman-installer/scenarios.d/satellite-answers.yaml --to-stdout
+  register: satellite_answers
+  ignore_errors: true
+
+- name: set satellite_version to 6.2 if satellite_answers.yaml exists
+  set_fact:
+    satellite_version: 6.2
```

Comment:
while you are at this, please also think about how to determine satellite version 6.3 as this will be added to the mix very soon.